lib/image-diff.js

```diff
@@ -2,16 +2,17 @@
 var fs = require('fs');
 var path = require('path');
 var async = require('async');
-var gm = require('gm').subClass({imageMagick: true});
+var gm;
```

Comment:
To do that, we should start accepting options in `function ImageDiff` and creating an instance of `gm` there.

```js
// Load in `gm` dependency via `require` at the top
var gm = require('gm');

// Inside of the constructor, we adjust `gm` as needed:
function ImageDiff(options) {
  var useImageMagick = options.imageMagick === undefined || options.imageMagick;
  this.gm = useImageMagick ? gm.subclass({imageMagick: true}) : gm;
}
```